lib/scenario_runner.py

```diff
@@ -1440,22 +1448,58 @@ def _setup_services(self):
                     'detach': cmd_obj.get('detach', False),
                 })
 
-        print(TerminalColors.HEADER, '\nCurrent known containers: ', self.__containers, TerminalColors.ENDC)
+        container_names = [container_info['name'] for container_info in self.__containers.values()]
+        print(TerminalColors.HEADER, '\nStarted containers: ', container_names, TerminalColors.ENDC)
 
-    def _add_to_current_run_log(self, identifier, message):
-        self.__current_run_logs.append(f"{identifier}\n{message}")
+    def _add_to_current_run_log(self, container_name, log_type, log_id, cmd, phase, stdout=None, stderr=None, flow=None, exception_class=None):
+        """Add a structured log entry to the current run logs.
+        
+        Args:
+            container_name: Name of the container or '[SYSTEM]' for system logs
+            log_type: LogType enum value
+            log_id: ID of the process (from id(ps) or id(exception))
+            cmd: Command that was executed
+            phase: Phase like '[BOOT]', '[RUNTIME]', etc.
+            stdout: Standard output (optional)
+            stderr: Standard error (optional)
+            flow: Flow name for flow_command type (optional)
+            exception_class: Exception class name for exception type (optional)
+        """
+        if container_name not in self.__current_run_logs:
+            self.__current_run_logs[container_name] = []
+
+        log_entry = {
+            "type": log_type.value,
```

Comment:
please adhere to our contribution.md style guide that all non-interpolated strings are single quoted. Unless  they contain single quotes themselves

Reply:
The file is full of double-quotes. Maybe this should be something for a separate PR to clean this up.
btw: In the `CONTRIBUTING.md` there is only a style guide for Python, not for JavaScript 😉